<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reconstructmeqt</class>
 <widget class="QMainWindow" name="reconstructmeqt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ReconstructMe</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <item>
     <widget class="QWidget" name="scanWidget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="play_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>40</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Toggle Play/Pause</string>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../rc/res.qrc">
               <normaloff>:/images/record-button.png</normaloff>:/images/record-button.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="reset_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>40</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Reset</string>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../rc/res.qrc">
               <normaloff>:/images/reset-button.png</normaloff>:/images/reset-button.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="scanPage">
            <layout class="QGridLayout" name="view_layout" columnstretch="2,1">
             <item row="0" column="1">
              <widget class="ReconstructMeGUI::QGLCanvas" name="rgb_canvas" native="true"/>
             </item>
             <item row="1" column="1">
              <widget class="ReconstructMeGUI::QGLCanvas" name="depth_canvas" native="true"/>
             </item>
             <item row="0" column="0" rowspan="2">
              <widget class="ReconstructMeGUI::QGLCanvas" name="rec_canvas" native="true"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="surfacePage">
            <layout class="QGridLayout" name="gridLayout_6" columnstretch="10,0">
             <item row="4" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="1" rowspan="2">
              <widget class="QGroupBox" name="groupBox_2">
               <property name="maximumSize">
                <size>
                 <width>220</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="title">
                <string>Surface</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="0" column="0" rowspan="2">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Number of Vertices</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Number of Triangles</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0" colspan="3">
                   <widget class="QPushButton" name="saveButton">
                    <property name="text">
                     <string>Save</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1" colspan="2">
                   <widget class="QSpinBox" name="numTrianglesLE">
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="maximum">
                     <number>999999999</number>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1" rowspan="2" colspan="2">
                   <widget class="QSpinBox" name="numVerticesLE">
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="maximum">
                     <number>999999999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QGroupBox" name="gbSurfaceProcessing">
               <property name="maximumSize">
                <size>
                 <width>220</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="title">
                <string>Number of Triangles</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QSlider" name="numTriangleSlider">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="singleStep">
                   <number>5</number>
                  </property>
                  <property name="pageStep">
                   <number>10</number>
                  </property>
                  <property name="value">
                   <number>100</number>
                  </property>
                  <property name="sliderPosition">
                   <number>100</number>
                  </property>
                  <property name="tracking">
                   <bool>false</bool>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="invertedAppearance">
                   <bool>false</bool>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::NoTicks</enum>
                  </property>
                  <property name="tickInterval">
                   <number>25</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="numTriangleSpinBox">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="wrapping">
                   <bool>false</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="keyboardTracking">
                   <bool>true</bool>
                  </property>
                  <property name="suffix">
                   <string>%</string>
                  </property>
                  <property name="prefix">
                   <string/>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="value">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0" rowspan="5">
              <widget class="ReconstructMeGUI::viewer_widget" name="viewer" native="true">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>200</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Render Mode</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="0">
                 <widget class="QRadioButton" name="polygonRB">
                  <property name="text">
                   <string>Fill</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QRadioButton" name="wireframeRB">
                  <property name="text">
                   <string>Wireframe</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionProjectHome"/>
    <addaction name="actionForum"/>
    <addaction name="actionDevice"/>
    <addaction name="actionFAQ_2"/>
    <addaction name="separator"/>
    <addaction name="actionGenerate_hardware_key"/>
    <addaction name="actionLog"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="reconstruct_satus_bar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionInstallation">
   <property name="text">
    <string>Installation and Hardware</string>
   </property>
  </action>
  <action name="actionDevice">
   <property name="text">
    <string>Online Device Compatibility Matrix</string>
   </property>
  </action>
  <action name="actionUsage">
   <property name="text">
    <string>Usage</string>
   </property>
  </action>
  <action name="actionFAQ">
   <property name="text">
    <string>FAQ</string>
   </property>
  </action>
  <action name="actionForum">
   <property name="text">
    <string>Online Forum</string>
   </property>
  </action>
  <action name="actionLoad_config">
   <property name="text">
    <string>load config</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save 3D model</string>
   </property>
  </action>
  <action name="actionGenerate_hardware_key">
   <property name="text">
    <string>Product Licensing</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="actionSDKDocumentation">
   <property name="text">
    <string>SDK Documentation</string>
   </property>
  </action>
  <action name="actionProjectHome">
   <property name="text">
    <string>Online Project Homepage</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
  </action>
  <action name="action3D_Preview">
   <property name="text">
    <string>3D Preview</string>
   </property>
  </action>
  <action name="actionFAQ_2">
   <property name="text">
    <string>Online FAQ</string>
   </property>
  </action>
  <action name="actionOpen_Volume">
   <property name="text">
    <string>Open Volume</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ReconstructMeGUI::viewer_widget</class>
   <extends>QWidget</extends>
   <header>osg_viewer_qt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReconstructMeGUI::QGLCanvas</class>
   <extends>QWidget</extends>
   <header>qglcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>reconstructmeqt</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numTriangleSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>numTriangleSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>852</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
